File: tuts/005-cloudfront-gettingstarted/javascript/scenarios/getting-started.js
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

import { CloudFrontClient } from "@aws-sdk/client-cloudfront";
import { randomUUID } from "crypto";

const main = async () => {
  const client = new CloudFrontClient({});
  const suffix = randomUUID().slice(0, 8);
  console.log("Running cloudfront getting started scenario...");
  // TODO: implement setup, interact, teardown
  console.log("Scenario complete.");
};

main();

File: tuts/005-cloudfront-gettingstarted/go/scenarios/getting_started.go
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

package main

import (
	"context"
	"fmt"
	"github.com/aws/aws-sdk-go-v2/config"
	aws_cloudfront "github.com/aws/aws-sdk-go-v2/service/cloudfront"
)

func main() {
	cfg, _ := config.LoadDefaultConfig(context.TODO())
	client := aws_cloudfront.NewFromConfig(cfg)
	_ = client
	fmt.Println("Running CloudFront getting started scenario...")
	// TODO: setup, interact, teardown
}
